src/main/java/net/sf/jsqlparser/statement/SetStatement.java

```diff
@@ -10,8 +10,8 @@
 package net.sf.jsqlparser.statement;
 
 import java.io.Serializable;
-import java.util.ArrayList;
-import java.util.List;
+import java.util.*;
```

Comment:
Using wildcard imports is a bad practice. More than needed is imported and could potentially link unneeded modules to the library.